rollup/default-with-browser.js

```diff
@@ -0,0 +1,142 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import path from 'path';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+
+export default [
+	{
+		input: 'src/index.js',
+		output: [
+			{ file: 'dist/index.cjs', format: 'cjs', sourcemap: true, exports: 'auto' },
+			{ file: 'dist/index.mjs', format: 'esm', sourcemap: true, exports: 'auto' },
+		],
+		external: [
+			'postcss-values-parser',
+			'postcss-selector-parser',
+		],
+		plugins: [
+			babel({
+				babelHelpers: 'bundled',
+				exclude: 'node_modules/**',
+				presets: [
+					['@babel/preset-env', {
+						corejs: 3,
+						loose: true,
+						modules: false,
+						targets: { node: 12 },
+						useBuiltIns: 'usage',
+					}],
+				],
+			}),
+			terser(),
+		],
+	},
+	{
+		input: 'src/browser.js',
+		output: [
+			{ file: 'dist/browser.cjs', format: 'cjs', sourcemap: true, exports: 'auto', strict: false },
+		],
+		plugins: [
+			babel({
+				babelHelpers: 'bundled',
+				exclude: 'node_modules/**',
+				presets: [
+					['@babel/preset-env', {
+						loose: true,
+						modules: false,
+						targets: {
+							browsers: [
+								'IE >= 8',
+								'Opera >= 12',
+								'Safari >= 5.1',
+								'Chrome >= 15',
+								'Edge >= 12',
+								'Firefox >= 4',
```

Comment:
This feels a bit too much for me.  This will bundle a bunch of polyfills that most browsers nowadays (MutationObserver is over 98% compatible). I think we could offer one fully polyfilled but another that's slimmer so the developer has the option to selectively polyfill.

Reply:
No polyfills are added.
Only the javascript itself is transpiled down to this list.

We can't include any actual polyfills as the chance that this will be the only script (and the only polyfill provider) is very slim. Having duplicate and conflicting polyfills is a very common source of issues.

I also documented the full list of features required to run this in any browser, so that users can provide these.

-----

98% is actually very low if you think of it in number of visitors that have a broken experience. 2 in 100 visitors would find a non functional page.

With the needed polyfills and transpiling down to this point it is close to 99.99% or 1 visitor in 10.000.

Reply:
I get that but my point was more as, give the choice to the developer. The number could be actually 100% on it's reduced use case as 98 is a global number from can I use. I saw the list and this other list made me think it would polyfill them, apologies!